modules/markup/console/console.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package console
+
+import (
+	"bytes"
+	"io"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"code.gitea.io/gitea/modules/markup"
+	"code.gitea.io/gitea/modules/setting"
+
+	trend "github.com/buildkite/terminal-to-html/v3"
+	"github.com/go-enry/go-enry/v2"
+)
+
+// MarkupName describes markup's name
+var MarkupName = "console"
```

Comment:
Does anything speak against
```suggestion
const MarkupName = "console"
```
?

Reply:
I can change but I was following how it's implemented in other markup renderers

Reply:
should I change it to const or leave as in other renderers?

Reply:
Okay, leave it as is.

Reply:
ok, thanks for review :)